### ISSUE TITLE
J2K transfer syntax support

### ISSUE THREAD

**Issue by arunjoshtech** [Enet4/dicom-rs#132]
Does project support j2k transfer syntax?

**Comment by Enet4** [comment]
Yes, the `pixeldata` crate from the DICOM-rs project is at an early stage, but it supports JPEG 2000 encodings via GDCM bindings. You can try it out with the upstream revision. You can also keep track of development around pixeldata here: #125
